pkg/permissionclaim/permissionclaim_labeler.go

```diff
@@ -43,6 +44,7 @@ type Labeler struct {
 func NewLabeler(apiBindingInformer apisv1alpha1informers.APIBindingClusterInformer) *Labeler {
 	return &Labeler{
 		listAPIBindingsAcceptingClaimedGroupResource: func(clusterName logicalcluster.Name, groupResource schema.GroupResource) ([]*apisv1alpha1.APIBinding, error) {
+			// TODO(nrb): do we need an indexer for name/ns? May get big
```

Comment:
In the future if we add label selectors or whatever other mechanism, we can't have a unique index per permuation of selection logic, so I think the list+filter approach here looks good.